docs/README.md

```diff
@@ -36,9 +36,12 @@
 - [SAT Kibana Dashboards](dashboards/SAT_Kibana_Dashboards.md)
 - [SAT Grafana Dashboards](dashboards/SAT_Grafana_Dashboards.md)
 
-## [SAT Usage](usage.md)
+## [SAT Bootprep](sat_bootprep.md)
 
-- [SAT Bootprep](usage.md#sat-bootprep)
+- [SAT bootprep vs SAT bootsys](sat_bootprep.md#sat-bootprep-vs-sat-bootsys)
+- [Editing a bootprep input file](sat_bootprep.md#editing-a-bootprep-input-file)
+- [Example bootprep input files](sat_bootprep.md#example-bootprep-input-files)
+- [Viewing built-in generated documentation](sat_bootprep.md#viewing-built-in-generated-documentation)
```

Comment:
I'm noticing that we generally use title-style capitalization for our section headings. However, there are a few places that we use sentence-style capitalization instead. For example, [Remove obsolete configuration file sections](install.md/#remove-obsolete-configuration-file-sections) and the headings within [SAT Release Notes](release_notes.md).

We should try to be consistent, so you might want to change your headings here and in the SAT bootprep file to title-style capitalization instead. I can try to update the other headings to title-style capitalization in the future.

Reply:
Sure, fixed with 1770352b652ea3d13b83cb85e755d088da1e666c